src/components/Alphabet/AlphabetPresentationSelected.js

```diff
@@ -121,36 +121,45 @@ function AlphabetPresentationSelected({
           </div>
         </div>
       )}
-      <div className="flex justify-center space-x-2">
-        <Link
+      <div className="flex flex-col justify-center space-x-2 max-w-96 mx-auto">
+        {/* <Link
```

Comment:
Did you mean to leave this Link/button commented out?

Reply:
oops! nope, I missed that one. Good catch